packages/icons/src/library/collab-comment.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/primitives';
+
+const collabComment = (
```

Comment:
We should probably call this `speechBubble` or something like that. More about the symbol itself, less about how it might be used. cc @jasmussen 

This is something we can easily split into another PR, by the way.

Reply:
New PR added https://github.com/WordPress/gutenberg/pull/65180